backend/api/Controllers/ParcelController.cs

```diff
@@ -155,6 +156,7 @@ public IActionResult UpdateParcel(int id, [FromBody] Model.ParcelModel model)
         [Produces("application/json")]
         [ProducesResponseType(typeof(Model.ParcelModel), 200)]
         [SwaggerOperation(Tags = new[] { "parcel" })]
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Style", "IDE0060:Remove unused parameter", Justification = "To support standardized routes (/delete/{id})")]
```

Comment:
Keeping the `id` for the route path throws a warning because we don't presently use it.  I've suppressed the warning.